policy/cnspec_policy.proto

```diff
@@ -33,16 +56,60 @@ message PolicySpec {
   int64 created = 32;
   int64 modified = 33;
 }
+// ^^
+
+message PolicyRef {
+  enum Action {
+    UNKNOWN = 0;
+    ADD = 1;
+    DISABLE = 2;
+    MODIFY = 3;
+  }
+
+  string mrn = 1;
+  string uid = 2;
+
+  repeated cnquery.explorer.Property props = 35;
+  cnquery.explorer.Filters filters = 43;
+  cnquery.explorer.Impact.ScoringSystem scoring_system = 10;
+  // Action is used for all policy overrides
+  Action action = 41;
+
+  string graph_content_checksum = 38;
+  string graph_execution_checksum = 40;
+}
 
 /*
   Policy definition, re-usable or assigned to the user's environment
   - Contains all policies and overrides
   - Can limit to what it supports
 */
 message Policy {
+  // FIXME: DEPRECATED, remove in v9.0 vv
+  // This is replaced by the field "filters"
+  map<string,Mquery> v1_asset_filters = 7; // key == checksum, internal
```

Comment:
I am not sure I missed it, do we have a custom unmarshaller to load old policies?